tracing/suite.go

```diff
@@ -48,50 +48,19 @@ func NewClientSuite(opts ...Option) *clientSuite {
 		clientOpts,
 		client.WithMiddleware(ClientMiddleware(cfg)),
 		client.WithTransportProtocol(transport.TTHeader),
-		client.WithMetaHandler(transmeta.ClientTTHeaderHandler),
-	}
-	return &clientSuite{cOpts}
-}
-
-func NewFramedClientSuite(opts ...Option) *clientSuite {
-	clientOpts, cfg := newClientOption(opts...)
-	cOpts := []client.Option{
-		clientOpts,
-		client.WithMiddleware(ClientMiddleware(cfg)),
-		client.WithTransportProtocol(transport.Framed),
-		client.WithMetaHandler(transmeta.ClientTTHeaderHandler),
-	}
-	return &clientSuite{cOpts}
-}
-
-func NewGRPCClientSuite(opts ...Option) *clientSuite {
```

Comment:
it is not suggested to remove directly, add a comment
`// Deprecated: Use NewClientSuite instead.`

Reply:
> it is not suggested to remove directly, add a comment `// Deprecated: Use NewClientSuite instead.`

Thanks for the reminder, corrected